lmcache/cache_engine.py

```diff
@@ -74,7 +66,7 @@ def _hash(
     def _chunk_tokens(
             self, 
             tokens: torch.Tensor, 
-            device
+            #device
```

Comment:
A general comment: just feel free to remove the useless code instead of commenting it out -- git can be used to track history anyway.